pkg/cmd/cli/describe/projectstatus.go

```diff
@@ -1259,6 +1263,53 @@ func (l *secretLoader) AddToGraph(g osgraph.Graph) error {
 	return nil
 }
 
+type pvcLoader struct {
+	namespace string
+	lister    kclient.PersistentVolumeClaimsNamespacer
+	items     []kapi.PersistentVolumeClaim
+}
+
+func (l *pvcLoader) Load() error {
+	list, err := l.lister.PersistentVolumeClaims(l.namespace).List(kapi.ListOptions{})
+	if err != nil {
+		return err
+	}
+
+	l.items = list.Items
+	return nil
+}
+
+func (l *pvcLoader) AddToGraph(g osgraph.Graph) error {
+	for i := range l.items {
+		kubegraph.EnsurePersistentVolumeClaimNode(g, &l.items[i])
+	}
+
+	return nil
+}
+
+type nodeLoader struct {
```

Comment:
> I am not exposing nodes in any way in the output. I just check if there are more than one in order to determine whether we will warn for RWO volumes. I cannot find any other way to do this in the marker level other than having nodes as graph nodes (because all we give to markers is a graph).

Right, but when a project-admin runs this command, he'll always get a 403 and an incomplete graph because he won't be able to do the "list nodes" operation.